### ISSUE TITLE
Support faster mouse selections

### ISSUE THREAD

**Issue by slimbuck** [playcanvas/pcui#106]
Context menus and select inputs require the user to click the mouse button twice to make a selection.

Ideally single click selections would also be possible, meaning selections are achieved by:

buttonDown->selection->buttonUp

**Comment by Maksims** [comment]
Indeed! It used to work like that for most select-fields in Editor before.
It only works like that for Camera selector in Viewport top-right corner. Although there is an issue with position of input fields when non-Perspective camera is selected. Should be fixed probably.
